test/SlowTests/Issues/RavenDB-18426.cs

```diff
@@ -0,0 +1,139 @@
+using FastTests;
+using System.Linq;
+using Raven.Client.Documents;
+using Raven.Client.Documents.Indexes;
+using Raven.Client.Documents.Queries;
+using Raven.Server.Documents.Indexes.Static.Extensions;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace SlowTests.Issues;
+
+public class RavenDB_18426 : RavenTestBase
+{
+    public RavenDB_18426(ITestOutputHelper output) : base(output)
+    {
+    }
+
+    [Fact]
+    public void ProjectionIdTest()
+    {
+        using (var store = GetDocumentStore())
+        {
+            new DocIndex().Execute(store);
+            var requestExecutor = store.GetRequestExecutor();
+            using (var session = store.OpenSession())
+            {
+                session.Store(new Doc { Id = "doc-1", IntVal = 1 });
+                session.SaveChanges();
+            }
+
+            Indexes.WaitForIndexing(store);
+
+            using (var session = store.OpenSession())
+            {
+                var reducedQuery = session.Query<DocView, DocIndex>()
```

Comment:
This is redundant (covered by the test below).

Can delete until line 41 (Assert.Single).